CHANGELOG.md

```diff
@@ -53,7 +56,8 @@ The format is based on [Keep a Changelog 1.0.0].
 - Initial tagged release
 
 [Unreleased]: https://github.com/nationalarchives/ds-caselaw-custom-api-client/compare/release-4.0.0...HEAD
```

Comment:
This should be [Unreleased]: https://github.com/nationalarchives/ds-caselaw-custom-api-client/compare/release-4.2.0...HEAD